Core/Battler.gd

```diff
@@ -66,8 +66,13 @@ func _on_enemy_start_turn() -> void:
 	
 	# enemy attack
 	for enemy: Entity in _enemy_list:
-		var success: bool = _on_attack(enemy.get_behavior_component().attack, enemy, PlayerManager.player)
-		assert(success == true, "Enemy failed to attack.")
+		var enemy_attack: CardBase = enemy.get_behavior_component().attack
```

Comment:
this seems to be out of the scope of the PR and the related issue

Reply:
Yeah true. This needed to change because I changed the way cards are played, which also affects enemy attacks since they use a CardBase to attack. I can revert the change if you’d like, we’ll have to fix enemy attacks in a different PR though

Reply:
leave it like that then